javascript/javascript3/week1/happy_project/assets/vendors/bootstrap/bootstrap.affix.js

```diff
@@ -0,0 +1,162 @@
+/* ========================================================================
+ * Bootstrap: affix.js v3.3.6
+ * http://getbootstrap.com/javascript/#affix
+ * ========================================================================
+ * Copyright 2011-2015 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ * ======================================================================== */
+
+
++function ($) {
```

Comment:
do u need this + sign

Reply:
Happy_project is created for us by default. We are just responsible for the scripts. I just implemented the functionality with the API.